tests/test_reader.py

```diff
@@ -206,7 +206,7 @@ class DeflateReaderIntegrationTest(ReaderIntegrationTest):
         'deflate_level': 6,
         'tls_v1': True,
         'tls_options': {'cert_reqs': ssl.CERT_NONE},
-        'heartbeat_interval': 10,
+        'heartbeat_interval': 1,
```

Comment:
If I remember correctly, it was for a faster test while debugging. Like it reverted?